extensions/shared/test/get-site-fragment.js

```diff
@@ -1,58 +0,0 @@
-/**
- * Internal dependencies
- */
-import getSiteFragment from '../get-site-fragment';
-
-describe( 'getSiteFragment()', () => {
-	let beforeWindow;
-
-	beforeAll( () => {
-		beforeWindow = global.window;
```

Comment:
This is a pain to get working with newer jsdom versions which errors because assignment to `window.location` implies navigation.

I don't think we lose much value dropping these tests.